hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -80,7 +81,6 @@ public class BackgroundPipelineCreator implements SCMService {
    * configured interval and tries to create pipelines.
    */
   private Thread thread;
-  private final Object monitor = new Object();
```

Comment:
I suggest to keep the private monitor field.  It can avoid code outside this class calling wait/notify/notifyAll.